materials/module-1/worksheets/3-community-local-impact.md

```diff
@@ -0,0 +1,64 @@
+Community & Local Impact
```

Comment:
I like these, but we should have a PDF version added too

Reply:
Eventually yes